src/UIKit/UIEnums.cs

```diff
@@ -2318,16 +2368,14 @@ public enum UIImageSymbolWeight : long
 
 	[Mac (10,15, onlyOn64: true), iOS (13,0), TV (13,0)]
 	[Native]
-	public enum NSTextScalingType : long
-	{
+	public enum NSTextScalingType : long {
```

Comment:
try not to fix style in the same commit that adds code
ease reviewing (now) and blaming (later)

Reply:
Yeah I agree but (🙃) I decided to fix this on the stuff that I included in my last PR it would not ease review but will still blame back to me